README.md

```diff
@@ -599,7 +599,15 @@ The following example was "contrived". The `drupal_loadtest` example was run for
  Aggregated               |      432.98 |     294.11 |       3,390 |         14
  ```
 
-From these two tables, it is clear that there was a statistically significant event affecting the load testing metrics. In particular, note that the standard deviation between the "raw" average and the "adjusted" average is considerably larger than the "raw" average, calling into questing whether or not your load test was "valid". (The answer to that question depends very much on your specific goals and load test.)
+Note: It is beyond the scope of Goose to test for statistically significant changes in the right-tail, or other locations, of the distribution of response times.  Goose produces the raw data you need to conduct these tests.
+
+Nonetheless, for users interested in establishing if there was an event(s) affecting the shape of the distribution of load test metrics (by a statistically significant amount): The following program is a reasonable starting point.
+
+1. Run a test without the CO event(s), record the raw response data.
+2. Run a test with the CO events, record the CO-adjusted response data.
```

Comment:
This doesn't offer much value, unfortunately, at least not in the sorts of load tests I run for clients. There's no specific "CO event" that I would intentionally trigger and then intentionally not trigger to compare the results. Instead, Goose is indeed attempting to automatically detect when it happens and to warn you: and further it attempts to back-fill the metrics with what it statistically expects it would have seen for the other requests during the event.